plugin/parser/transform/mysql/transform_test.go

```diff
@@ -0,0 +1,27 @@
+package transform
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateTableSeparateIndex(t *testing.T) {
+	input := "CREATE TABLE `t1` (\n" +
+		"	`id` int DEFAULT NULL,\n" +
+		"	`c1` int DEFAULT NULL,\n" +
+		"	`c2` int DEFAULT NULL,\n" +
+		"	UNIQUE KEY `c1` (`c1`,`c2`),\n" +
+		"	UNIQUE KEY `haha` (`c2`),\n" +
+		"	KEY `t1` (`c1`)\n" +
+		") ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;"
+	want := "CREATE TABLE `t1` (`id` INT DEFAULT NULL,`c1` INT DEFAULT NULL,`c2` INT DEFAULT NULL) ENGINE = InnoDB DEFAULT CHARACTER SET = UTF8MB4 DEFAULT COLLATE = UTF8MB4_0900_AI_CI;\n" +
+		"CREATE UNIQUE INDEX `c1` ON `t1` (`c1`, `c2`);\n" +
+		"CREATE UNIQUE INDEX `haha` ON `t1` (`c2`);\n" +
+		"CREATE INDEX `t1` ON `t1` (`c1`);\n"
```

Comment:
TBH this format is pretty hard to read and parse in the head... but doesn't seem like there is a super handy approach for MySQL with backticks.

Reply:
The closest attempt I had was https://github.com/flamego/session/blob/2b1f2c612af2d8d95a22b8494cd437525f8edc3f/mysql/mysql.go#L62-L64 but still not so great

Reply:
Looks like something actually better than current 🤔 👍